### PR TITLE
[*] MEOW : Meow Meow Meow

### PR DESCRIPTION
**Pull request by PuffDaddyCat**
Meow !

### CODE REVIEW

**Comment by PuffDaddyCat** [comment]
:feet: MEOW !

File: index.php [hunk 1]
```diff
@@ -24,5 +24,7 @@
 *  International Registered Trademark & Property of PrestaShop SA
 */
 
+// Meow !
```
Comment (Quetzacoalt91) [inline index.php]: Regarding the LOLCODE (http://en.wikipedia.org/wiki/LOLCODE#Example_2), you should write ```BTW "Meow !"```

Comment (PuffDaddyCat) [inline index.php]: Meow ?

**Comment by gRoussac** [comment]
:joy_cat:

**Pull request closed without merging**